app/code/Magento/Contact/Controller/Index/Post.php

```diff
@@ -70,8 +79,10 @@ public function execute()
             $this->messageManager->addErrorMessage($e->getMessage());
             $this->getDataPersistor()->set('contact_us', $this->getRequest()->getParams());
         } catch (\Exception $e) {
+            $this->logger->addError('An error occurred in Controller\\Index\\Post: ' . $e->getMessage());
```

Comment:
a) why do we need this one? wouldn't the stack trace by enough? 
b) if you're going to do it, why aren't you using `__CLASS__`?